### ISSUE TITLE
Compile and Render fails with imported STL (tardis)

### ISSUE THREAD

**Issue by mattlyle** [openscad/openscad#453]
I am trying to import an STL file and integrate it into a larger model... when I preview everything, it works great, but when I compile and render, it thinks the imported item doesn't exist anymore.  There are no errors given in the window.  I have tested with the 2013.06 for Windows and Linux and they fail in the identical manner.

This is a large STL file (36 MB), so I can't even attach it here, but the file is located here:
http://www.thingiverse.com/thing:81182

Basically I am just trying to do this:
import( "actual_tardis.stl" );

translate( [ -100, -100, 0 ] )
    cube( [ 200, 200, 0.5 ] );

Update Feb 2016:
- The 3 models posted here (`actual_tardis.stl, bulletsfinalpuzzle.stl, bulletsfinalpuzzle_Cut_1.stl`) all contain unknown issues and still causes import to fail
- The repaired tardis `actualtardis_fixed.stl` works fine
##

**Comment by kintel** [comment]
Status as of 2024-03-24: All the example in this issue now works using Manifold:
* https://www.thingiverse.com/thing:81182 actual_tardis.stl
* https://www.thingiverse.com/thing:81182 actual_tardis_fixed.stl
* http://www.thingiverse.com/thing:43672
